docs/source/tutorials/playing-command.rst

```diff
@@ -0,0 +1,129 @@
+=========================
+Playing with your command
+=========================
+
+Now that you have started :doc:`first-plugin`, maybe you want to write a more
+interesting command than the basic ``.hello`` one. Not that there is anything
+wrong with that command! The emoticons plugin is composed of commands like this
+one::
+
+    @plugin.command('shrug')
+    @plugin.action_command('shrugs')
+    def shrug(bot, trigger):
+        bot.say('¯\\_(ツ)_/¯')
+
+Which is one of the maintainers' favorite command to use. However, let's see
+if we can do a bit more complex than that.
+
+Greeting a user by their name
+=============================
+
+Have you noticed that a :func:`plugin_callable` takes two arguments? The first
+one is the ``bot``, i.e. an instance of Sopel (wrapped for your convenience)
+that you can use to :doc:`/plugin/bot`.
+
+In the previous tutorial, we used ``bot.reply``, which is convenient but not
+always what you want. Maybe you want the bot to say this::
+
+    <YourNick> .hello
+    <Sopel> Hello YourNick, have a nice day!
+
+For that, you need the second argument: the ``trigger``. It is an object with
+all the information you need about the message that triggered the callable,
+such as the message itself, the channel, the type of message, etc. and what we
+need for now is the :attr:`trigger.nick <sopel.trigger.Trigger.nick>`
+attribute::
+
+    from sopel import plugin
+
+    @plugin.command('hello')
+    def hello(bot, trigger):
+        """Say Hello <user>, have a nice day!"""
+        bot.say(f'Hello {trigger.nick}, have a nice day!')
+
+.. seealso::
+
+    You can learn much more about the :class:`trigger <sopel.trigger.Trigger>`
+    object by reading its documentation.
+
+Command with arguments
+======================
+
+The trigger object can do much more for you: if a user add arguments to the
+command, like this: ``.hello morning``, you can detect and use that argument::
```

Comment:
```suggestion
command, like ``.hello morning``, you can detect and use that argument::
```

This is for flow. Colon followed by comma is a bit odd.

(Would be nice if it also puts `.hello morning` on the same line, but I doubt deleting `this: ` will free up enough space in Furo.)